flink-runtime/src/test/java/org/apache/flink/runtime/operators/testutils/MockEnvironment.java

```diff
@@ -335,7 +335,7 @@ public void acknowledgeCheckpoint(long checkpointId, CheckpointMetrics checkpoin
 
 	@Override
 	public void declineCheckpoint(long checkpointId, Throwable cause) {
-		throw new UnsupportedOperationException();
+		throw new UnsupportedOperationException(cause);
```

Comment:
nit: unrelated change for this commit motivation

Reply:
Agree, but I don't think this change deserves its own commit.

Reply:
Based on my experience, any unrelated tiny changes should be submitted as `hotfix` commit instead, such as typo, indentation formatting, etc. 
But I guess there are no explicit guidelines for it, so feel free to do it or not.